webshopping/lolshopping/templates/design/home.html

```diff
@@ -45,12 +46,14 @@
 
 </div>
 
-    <div class="row">
+    <div class="row" id="results">
+
         {% for champs in all %}
         <div class="col-lg-3 abc">
             <img class ="thumbnail" src="{{champs.imageURL}}">
             <div class="box-element product">
-                <h6>{{champs.name}}</h6>
+                <div class="namez">
```

Comment:
Why does the class name have "z" in it?

Reply:
Can't think of a name, just needed for reference. but I can change it